src/Controllers/QuestionTemplateController.php

```diff
@@ -73,6 +123,13 @@ public function deleteQuestion(Request $request, array $requestAttributes)
         return new Response($body, '1.1', '301', $location);
     }
 
+    /**
+     * This method returns a Response with the attributes of a question fro autocomplete them in the edit form
```

Comment:
Maybe something like "Displays a pre-filled 'Edit Question' form"?
(Not sure that's what the form is actually called.)